tools/isobuild/meteor-npm.js

```diff
@@ -777,7 +777,7 @@ Profile("meteorNpm.runNpmCommand", function (args, cwd) {
           });
         }
       );
-    });
+    }).await();
```

Comment:
Yes, they should be identical. I'm not sure why (or even if) this commit changes anything, but I cherry-picked it from `devel` because I trust that @abernix observed a benefit.